iceoryx_posh/source/roudi/port_manager.cpp

```diff
@@ -659,19 +659,43 @@ popo::InterfacePortData* PortManager::acquireInterfacePortData(capro::Interfaces
     }
 }
 
+void PortManager::publishCurrentServiceRegistry() const noexcept
+{
+    // Send the new serviceRegistry here and ring the bell to inform all ServiceDiscovery instances about the change
```

Comment:
Please remove the comment, the method is simple enough and in the end you rephrased the method name with some lyrical finesse ;)